lms/djangoapps/discussion/django_comment_client/tests/test_utils.py

```diff
@@ -1003,8 +1000,7 @@ def test_ids_configured_topics(self):
             "Topic B": {"id": "Topic_B"},
             "Topic C": {"id": "Topic_C"}
         }
-        six.assertCountEqual(
-            self,
+        self.assertCountEqual(
```

Comment:
I know it's pretty late to be chiming in on this PR, but note that `assertCountEqual()` does more than its name implies; it makes sure that each side of the comparison contains exactly the same items, but not necessarily in the same order:  https://docs.python.org/3.8/library/unittest.html#unittest.TestCase.assertCountEqual

I'm not sure there's an easy equivalent to this with `assert`, so this may be a case where we want to keep the original method despite the poor naming.